test/sql/function/timestamp/test_try_strptime.test

```diff
@@ -90,9 +90,9 @@ SELECT try_strptime('30', '%U'), strftime('1900-07-29'::DATE, '%U')
 1900-07-29 00:00:00	30
 
 query II
-SELECT try_strptime('30', '%W'), strftime('1900-07-29'::DATE, '%W')
+SELECT try_strptime('30', '%W'), strftime('1900-07-23'::DATE, '%W')
```

Comment:
Can we also add a test with `%U` (which results in `WEEK_NUMBER_PADDED_SUN_FIRST`)?

Reply:
The line above?

Reply:
The date changed because we are now using the first day of the week as a weekday default instead of Sunday. It actually tests the change!

Reply:
Oh yeah, makes sense